articles/provider-quantinuum.md

```diff
@@ -49,39 +47,58 @@ We recommend that users first validate their code using our API Validator. This
-
+Billing information: H1 Emulator usage is offered free-of-charge with a hardware subscription. For details, see Billing information for the [System Model H1](#system-model-h1).
 
-## System Model H1, Powered by Honeywell
+## System Model H1
 
-Quantinuum's quantum computer, the System Model H1, Powered by Honeywell, includes two hardware machine targets: H1-1 and H1-2.  Both machines have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [API Validator](#api-validator) prior to submitting them to the target machines.  
+Quantinuum's quantum computer, the System Model H1, Powered by Honeywell, includes two hardware machine targets: H1-1 and H1-2.  Both machines have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [Syntax Checker](#syntax-checker) prior to submitting them to the target machines.  
 
-Users may submit jobs to a specific machine (H1-1 or H1-2), or submit them to the *machine family*.  Submission to the machine family enables the submitted job to run on the first available, compatible machine. The only condition for compatibility is the number of qubits. If a user submits a job to a specific machine that is not available, the job will remain in that machine's queue until the machine becomes available.   
+Users may submit jobs to a specific machine (H1-1 or H1-2), or submit them to the *machine family*.  Submission to the machine family enables the submitted job to run on the first available, compatible machine. The only condition for compatibility is the number of qubits. If a user submits a job to a specific machine that is not available, the job will remain in that machine's queue until the machine becomes available.
 
 Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout their product lifecycle. Users are given access to the most up-to-date, advanced, and capable hardware available.
 
-
 - Job type: `Quantum Program`
 - Data Format: `honeywell.openqasm.v1`
 - Target ID:
-    - H1-1: `quantinuum.hqs-lt-s1` 
-    - H1-2: `quantinuum.hqs-lt-s2`
-    - H1 machine family: `quantinuum.hqs-lt`
+  - H1-1: `quantinuum.hqs-lt-s1` 
+  - H1-2: `quantinuum.hqs-lt-s2`
+  - H1 machine family: `quantinuum.hqs-lt`
 - Target Execution Profile: [Basic Measurement Feedback](xref:microsoft.quantum.target-profiles)
 
+Billing information:
```

Comment:
The billing information has been moved to the new [pricing page](https://docs.microsoft.com/en-us/azure/quantum/pricing?tabs=tabid-AQcredits%2Ctabid-AQcreditsQ%2Ctabid-payasgo&pivots=ide-computing#quantinuum), based on SEO and user needs. We try to keep providers' pages for technical details about the solvers.

Reply:
Hi Sonia! There was a delay in my reply since I was out of office. I sent a note to the Quantum Partner Help before creating this PR. I never got a reply back. I wanted to verify this is the right location to edit the pages since I noted a difference in the Microsoft pages vs. what is on Github. I assumed that was because what is on Github is ahead of what is on the Microsoft docs pages (https://docs.microsoft.com/en-us/azure/quantum/provider-quantinuum). I went off of what the latest version of the provider pages were here. There are a few things that need to be updated. For example, the Quantinuum Emulator does not get a "Provided by Honeywell" after it since it isn't a quantum device. Is there a different version/latest version I should edit? The pricing is there only because that is what was there in the one I started with. I didn't edit anything related to that. If you want to discuss offline I'm happy to do that or I can send another note to the Quantum Partner Help email. Thanks!!

Reply:
Hi!! Yes, this is the location for docs updates. Sometimes there might be some difference between Microsoft docs site and GitHub site because synchronization happens twice a day, at 10 am and 3 pm PT.

You can do all the changes you consider need to be done, and someone from the docs writer team will review your PR (in this case, myself :) ). If the billing info was still in the file you have edited, maybe it is because you started editing before that change was done, so your branch was not up to date. Don´t worry, changing the billing from here to the Pricing page was a new feature implemented this past month.

Besides that, the updates you made have no conflicts and I guess they need to be live asap. So if you are ok, I can remove the billing information to match current Quantinuum provider reference page, and merge the rest into main. 

Let me know if that works :)

Reply:
Hi Sonia! yes, that does work. That'd be great if you can do that and I'll pull down the latest, etc. to start fresh next time I go to make updates. 

Its true that I had started editing about a month ago and also hadn't been sure I was following the right process for merging back in. Is it correct to Fork the repo and do a PR this way? I assumed so since I am not added as a contributor on the repo.

Reply:
Yes, doing a fork is the right way to do it. It might happen sometimes that your branch is outdated if there has been any recent change. To avoid this, if you are a git user you can fetch to the main branch. Otherwise, don't worry, GitHub has tools to do it when doing the PR and the writer team will take care of that.

Reply:
Ok, sounds good! Good to know for the future. Feel free to make the necessary edits so everything is in the right place/on the right pages. :) I'll follow this process in the future.